### ISSUE TITLE
Failed to execute unzip

### ISSUE THREAD

**Issue by fridzema** [WsdlToPhp/PackageGenerator#166]
Hello, 

Anyone has an idea what this means? 

```
  - Updating wsdltophp/packagegenerator (3.0.3 => 3.0.4): Loading from cache    Failed to execute unzip -qq  '/Users/rf/workspace/pmp/vendor/wsdltophp/packagegenerator/fe74a9f75cc1acfe86690cb4b72ec034' -d '/Users/rf/workspace/pmp/vendor/composer/08c2d881'

error:  cannot create /vendor/composer/08c2d881/WsdlToPhp-PackageGenerator-ab2db90/tests/resources/generated/Valid?????????????.php
        Illegal byte sequence

    The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
    Unzip with unzip command failed, falling back to ZipArchive class
```

**Comment by mikaelcom** [comment]
Any progress?

**Comment by binarious** [comment]
Same issue for me. Removing `vendor` doesn't work. I'm unable to install this package via composer. I don't think using cyrillic characters in the filename is a good idea.

**Comment by mikaelcom** [comment]
Are you on Windows ? Mac ? Or Unix ?

**Comment by binarious** [comment]
macOs: 10.14.6, composer: 1.9.0, php: 7.3.7

**Comment by mikaelcom** [comment]
@Gemorroj, any though on this matter ? If I recall right, you're using cyrillic characters :)

**Comment by binarious** [comment]
The test files aren't required in the composer archive, are they? They could still exist in this repository to run the test suite, but be [excluded for composer consumers](https://getcomposer.org/doc/04-schema.md#archive). Just an idea.

**Comment by Gemorroj** [comment]
#50 
Yes, the use of Cyrillic in names is considered very bad form.
Perhaps this bug should be addressed to https://github.com/composer/composer

**Comment by mikaelcom** [comment]
> #50
> Yes, the use of Cyrillic in names is considered very bad form.
> Perhaps this bug should be addressed to https://github.com/composer/composer

Ok so I should rename the files containing Cyrillic to _English_ names :)

**Comment by binarious** [comment]
> Perhaps this bug should be addressed to https://github.com/composer/composer

It's seems to be an issue in unzip:
- https://github.com/adamhathcock/sharpcompress/issues/315
- https://unix.stackexchange.com/questions/251969/how-can-i-correctly-decompress-a-zip-archive-of-files-with-hebrew-names

**Comment by mikaelcom** [comment]
fixed in [3.2.3](https://github.com/WsdlToPhp/PackageGenerator/releases/tag/3.2.3) and [2.11.3](https://github.com/WsdlToPhp/PackageGenerator/releases/tag/2.11.3) :wink:
